content/en/get-started/deploy-your-app.md

```diff
@@ -1,59 +1,105 @@
+* You have an [Armory RNA installed]({{< ref "remote-network-agent/_index.md" >}}) in your cluster.
 * You have created the Kubernetes manifests for your web app.
 * You have two versions of your app to deploy.
 
-For this guide, you can use the Remote Network Agent that you deployed as part of the Quickstart, or you can [install a new one]({{< ref "remote-network-agent/_index.md" >}}). Instead of deploying Armory's sample app, you deploy your own web app using the CD-as-a-service deployment configuration from the Quickstart. 
 
 ### Directory structure
 
-This guide assumes the following directory structure:
+In this guide you will create a deployment config and two simple namespace configs. The namespace manifests should be in a `manifests` directory along with the Kubernetes manifests for deploying your app.
+
+The directory structure should look like this:
 
 ```
-<my-app>
+<your-app>
 ├── deployment.yaml
 └── manifests
-    ├── <my-app>-service.yaml
-    ├── <my-app>.yaml
+    ├── <your-app-service>.yaml
+    ├── <your-app>.yaml
     ├── namespace-staging.yaml
     └── namespace-prod.yaml
 ```
 
-The Kubernetes manifests for deploying your app are in the `manifests` directory. Replace `<my-app>` with the name of your app.
 
-Create your staging namespace in `namespace-staging.yaml` and your prod namespace in `namespace-prod.yaml`.  The deployment config file supports deploying manifests to specific targets, and the deployment config file uses separate namespace manifests to illustrate this feature.
 
-## Create your deployment config file
+## 1. Create a deployment config file
 
-This deployment config defines the following strategies:
+First create two manifests for the staging and prod namespaces. These are where you'll deploy your app and this also showcases the ability to deploy manifests to specific targets. Save these to the `manifests` directory.
 
-* `rolling`: deploy 100% of the app (staging)
-* `trafficSplit`: 75% to the current version and 25% to the new version (prod)
 
-In addition, prod deployment requires a manual approval to begin deployment and another to continue deployment after the traffic split.
+<div style="display: flex; gap: 5%; margin-bottom: 7px;">
+<div>
+<p style="font-weight: bold; margin-bottom: 0;"><code>staging-namespace.yaml</code></p>
+
+```yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: <your-staging-namespace>
+```
+</div>
+<div>
+<p style="font-weight: bold; margin-bottom: 0;"><code>prod-namespace.yaml</code></p>
+
+```yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: <your-prod-namespace>
+```
+</div>
+</div>
+
+
+<table>
+<tr>
+<th>namespace-staging.yaml</th>
+<th>namespace-prod.yaml</th>
+</tr>
+<tr>
+<td>
+<pre>
+<code >
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: &#x3C;your-staging-namespace&#x3E;
+</code>
+</pre>
+</td>
+<td>
+<pre>
+<code>
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: &#x3C;your-prod-namespace&#x3E;
+</code>
+</pre>
+</td>
+</tr>
+</table>
```

Comment:
```suggestion
{{< cardpane >}}
{{< card code=true header="namespace-staging.yaml" lang="yaml">}}
apiVersion: v1
kind: Namespace
metadata:
  name: <your-staging-namespace>
{{< /card >}}
{{< card code=true header="namespace-prod.yaml" lang="yaml" >}}
apiVersion: v1
kind: Namespace
metadata:
  name: <your-prod-namespace>
{{< /card >}}
{{< /cardpane >}}
```

Suggest replacing with a cardpane and cards. The card content has the code highlighting and copy button (as does original table with the embedded code blocks). Using the `<pre><code>` approach doesn't compile into a copiable code block.

![image](https://github.com/armory/docs-cdaas/assets/26799583/0d00acbe-8f58-4eab-91c3-e2a8a5af4f14)

Reply:
@KathrynLewis